opencart 2.2.0.0/upload/catalog/controller/smailyforopencart/cron_customers.php

```diff
@@ -72,6 +72,11 @@ public function index() {
                     die('Error with request to Smaily API, try again later.');
                 }
             }
+            $this->model_smailyforopencart_helper->editSettingValue(
+                'module_smaily_for_opencart_sync',
```

Comment:
2.2 version does not use `module_` prefix for settings. Maybe you can use `smaily_for_opencart` group and `smaily_for_opencart_sync_time` key to store the last synchronization time?